models/issue_label.go

```diff
@@ -359,6 +366,9 @@ func updateLabel(e Engine, l *Label) error {
 
 // UpdateLabel updates label information.
 func UpdateLabel(l *Label) error {
+	if !LabelColorPattern.MatchString(l.Color) {
```

Comment:
Validation here is not required since `l.Color` is not used or related (not even in `updateLabel`).

Reply:
@guillep2k updateLabel Do set colors:
https://github.com/go-gitea/gitea/pull/10129/files/730d282fd30251c23a4c3a555c2ab46fd850a86b#diff-0186cfcb4351665688ca92eaa8c6dd2dR363